pipeline.py

```diff
@@ -1,20 +1,110 @@
 import click
 import pandas as pd
+from typing import List, Dict, Tuple
 
 # Make sure all global variables match the form.
 SID = 'Student ID Number'
 NAME = 'Name (First and Last)'
 EMAIL = 'Email'
-
+FIRST_PREF = 'Section Preference'
+SECOND_PREF = 'Second Choice Section (optional)'
+FOUR = 'Friday 4:00 to 5:00 PM'
+FIVE = 'Friday 5:00 to 6:00 PM'
+SIX = 'Friday 6:00 to 7:00 PM'
+PLAYERS_PER_TIMESLOT = 16
 
 def valid_input(input_df: pd.DataFrame) -> bool:
     # Validates input file for all files.
     return SID in input_df and NAME in input_df and EMAIL in input_df
 
+# Helper method to add student with SID and preference TIME to list of players
+# who could possibly play at each time slot
+def append_time_group(sid: str, time: str, four: List[str],
```

Comment:
can you add a docstring since the return object is not obvious what we are returning